tests/suites/kafka_kerberos/kafka_kerberos_test.go

```diff
@@ -0,0 +1,73 @@
+package kafka_kerberos
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/mesosphere/kudo-kafka-operator/tests/suites"
+
+	"github.com/mesosphere/kudo-kafka-operator/tests/utils"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+)
+
+var customNamespace = "kerberos-ns"
+var krb5Client = &utils.KDCClient{}
```

Comment:
you can join them in a var block and initialize the krb5Client with variables we already know here: 
```
var (
	customNamespace = "kerberos-ns"
	krb5Client = &utils.KDCClient{
		Namespace: customNamespace,
	}
)
```

Reply:
PR updated